test/server/schemas/address-test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var assert = require('assert');
+var validator = require('server/schema-validator');
+var ripple = require('ripple-lib');
+
+
+suite('valid addresses', function() {
+  test('generate new wallet', function() {
+    var address = ripple.Wallet.generate().address;
```

Comment:
I think it would be better to make the test deterministic by hard-coding an address